packages/openactive-integration-tests/test/features/criteria-requirements.json

```diff
@@ -108,8 +108,9 @@
     },
     "customer-requested-cancellation": {
       "primary": {
-        "TestOpportunityBookableCancellable": 8,
-        "TestOpportunityBookableNotCancellable": 1
+        "TestOpportunityBookableCancellable": 12,
+        "TestOpportunityBookableNotCancellable": 1,
+        "TestOpportunityBookable": 2
```

Comment:
Note to self: Why has this changed?

Reply:
I think it's because of the two tests for which `skipMultiple` was removed, which both involve `TestOpportunityBookableCancellable` and `TestOpportunityBookable` items. Though I don't know where the `TestOpportunityBookableNotCancellable` has come from

Reply:
Re `TestOpportunityBookableNotCancellable`, it seems like it should have been in the criteria-requirements.json in master - it was already in the generated `.md` doc for customer-requested-cancellation. Maybe criteria-requirements.json hadn't been auto-generated in master?